breadcrumbs for news, events, services, and faqs [fixes #113]

abanuog opened the pull request:
added easy_breadcrumbs module as well.

brharp commented:
Is this testable?

-----Original Message-----
From: abanuog [mailto:notifications@github.com] 
Sent: Thursday, June 23, 2016 2:03 PM
To: ccswbs/hjckrrh <hjckrrh@noreply.github.com>
Subject: [ccswbs/hjckrrh] breadcrumbs for news, events, services, and faqs [fixes #113] (#483)

added easy_breadcrumbs module as well. 

________________________________


You can view, comment on, or merge this pull request online at:


  https://github.com/ccswbs/hjckrrh/pull/483


Commit Summary


*	added breadcrumb patterns for news, events, services, and faqs [fixes #113]


File Changes


*	D profiles/ug/modules/ug/ug_breadcrumbs/ug_breadcrumbs.features.menu_custom.inc <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-0>  (26) 
*	D profiles/ug/modules/ug/ug_breadcrumbs/ug_breadcrumbs.features.user_permission.inc <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-1>  (21) 
*	M profiles/ug/modules/ug/ug_breadcrumbs/ug_breadcrumbs.info <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-2>  (34) 
*	M profiles/ug/modules/ug/ug_breadcrumbs/ug_breadcrumbs.module <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-3>  (2) 
*	M profiles/ug/modules/ug/ug_breadcrumbs/ug_breadcrumbs.strongarm.inc <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-4>  (144) 
*	A sites/all/modules/easy_breadcrumb/LICENSE.txt <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-5>  (339) 
*	A sites/all/modules/easy_breadcrumb/README.txt <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-6>  (54) 
*	A sites/all/modules/easy_breadcrumb/easy_breadcrumb.info <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-7>  (16) 
*	A sites/all/modules/easy_breadcrumb/easy_breadcrumb.install <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-8>  (25) 
*	A sites/all/modules/easy_breadcrumb/easy_breadcrumb.module <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-9>  (172) 
*	A sites/all/modules/easy_breadcrumb/includes/EasyBreadcrumbConstants.inc <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-10>  (106) 
*	A sites/all/modules/easy_breadcrumb/includes/easy_breadcrumb.admin.inc <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-11>  (183) 
*	A sites/all/modules/easy_breadcrumb/includes/easy_breadcrumb.blocks.inc <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-12>  (294) 
*	A sites/all/modules/easy_breadcrumb/js/easy_breadcrumb.admin.js <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-13>  (33) 


Patch Links:


*	https://github.com/ccswbs/hjckrrh/pull/483.patch
*	https://github.com/ccswbs/hjckrrh/pull/483.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub <https://github.com/ccswbs/hjckrrh/pull/483> , or mute the thread <https://github.com/notifications/unsubscribe/AB8WnZWUPjddCD8w3MZas4IqOaM2iDb5ks5qOso9gaJpZM4I9E-Q> . <https://github.com/notifications/beacon/AB8WnYACYzjKSE-8HURA0reFprGs3kkFks5qOso9gaJpZM4I9E-Q.gif>

abanuog commented:
yes, it should be.

----- Original Message -----
From: "M. Brent Harp" <notifications@github.com>
To: "ccswbs/hjckrrh" <hjckrrh@noreply.github.com>
Cc: "abanuog" <abanerji@uoguelph.ca>, "Author" <author@noreply.github.com>
Sent: Friday, June 24, 2016 9:25:18 AM
Subject: Re: [ccswbs/hjckrrh] breadcrumbs for news, events, services, and faqs [fixes #113] (#483)

Is this testable?

-----Original Message-----
From: abanuog [mailto:notifications@github.com] 
Sent: Thursday, June 23, 2016 2:03 PM
To: ccswbs/hjckrrh <hjckrrh@noreply.github.com>
Subject: [ccswbs/hjckrrh] breadcrumbs for news, events, services, and faqs [fixes #113] (#483)

added easy_breadcrumbs module as well. 

________________________________


You can view, comment on, or merge this pull request online at:


  https://github.com/ccswbs/hjckrrh/pull/483


Commit Summary


*	added breadcrumb patterns for news, events, services, and faqs [fixes #113]


File Changes


*	D profiles/ug/modules/ug/ug_breadcrumbs/ug_breadcrumbs.features.menu_custom.inc <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-0>  (26) 
*	D profiles/ug/modules/ug/ug_breadcrumbs/ug_breadcrumbs.features.user_permission.inc <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-1>  (21) 
*	M profiles/ug/modules/ug/ug_breadcrumbs/ug_breadcrumbs.info <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-2>  (34) 
*	M profiles/ug/modules/ug/ug_breadcrumbs/ug_breadcrumbs.module <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-3>  (2) 
*	M profiles/ug/modules/ug/ug_breadcrumbs/ug_breadcrumbs.strongarm.inc <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-4>  (144) 
*	A sites/all/modules/easy_breadcrumb/LICENSE.txt <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-5>  (339) 
*	A sites/all/modules/easy_breadcrumb/README.txt <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-6>  (54) 
*	A sites/all/modules/easy_breadcrumb/easy_breadcrumb.info <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-7>  (16) 
*	A sites/all/modules/easy_breadcrumb/easy_breadcrumb.install <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-8>  (25) 
*	A sites/all/modules/easy_breadcrumb/easy_breadcrumb.module <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-9>  (172) 
*	A sites/all/modules/easy_breadcrumb/includes/EasyBreadcrumbConstants.inc <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-10>  (106) 
*	A sites/all/modules/easy_breadcrumb/includes/easy_breadcrumb.admin.inc <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-11>  (183) 
*	A sites/all/modules/easy_breadcrumb/includes/easy_breadcrumb.blocks.inc <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-12>  (294) 
*	A sites/all/modules/easy_breadcrumb/js/easy_breadcrumb.admin.js <https://github.com/ccswbs/hjckrrh/pull/483/files#diff-13>  (33) 


Patch Links:


*	https://github.com/ccswbs/hjckrrh/pull/483.patch
*	https://github.com/ccswbs/hjckrrh/pull/483.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub <https://github.com/ccswbs/hjckrrh/pull/483> , or mute the thread <https://github.com/notifications/unsubscribe/AB8WnZWUPjddCD8w3MZas4IqOaM2iDb5ks5qOso9gaJpZM4I9E-Q> . <https://github.com/notifications/beacon/AB8WnYACYzjKSE-8HURA0reFprGs3kkFks5qOso9gaJpZM4I9E-Q.gif> 




---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/ccswbs/hjckrrh/pull/483#issuecomment-228344479

abanuog commented:
closing this pull request, as we have a new breadcrumbs solution.
